src/pages/_app.tsx

```diff
@@ -51,7 +49,6 @@ const InitApp = (): null => {
   useTxPendingStatuses()
   useTxTracking()
   useBeamer()
-  useABTesting(AbTest.SAFE_CREATION)
```

Comment:
Please also remove `SAFE_CREATION` from the enum.

Reply:
I've removed it and added a comment so we know where to put future abTest entries